### PR TITLE
Holly iss 1602 ecosystem tests properties

### PR DESCRIPTION
**Pull request by Wyvinar**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: build-locally.sh [hunk 1]
```diff
@@ -133,6 +133,8 @@ function download_dependencies {
     if [[ "${build_type}" == "clean" ]]; then
         h2 "Cleaning the dependencies out..."
         rm -fr build/dependencies
+        h2 "Tidying go mod file..."
+        go mod tidy
```
Comment (techcobweb) [inline build-locally.sh:137]: might be good to check the return code for failure ?

File: test-galasactl-ecosystem.sh [hunk 2]
```diff
@@ -791,7 +791,546 @@ function launch_test_from_unknown_portfolio {
...
+    properties_get_with_namespace
+    properties_get_with_name
+    properties_get_with_prefix
+    properties_get_with_suffix
+    properties_get_with_infix
+    properties_get_with_prefix_infix_and_suffix
+    #properties_get_with_name_prefix_infix_and_suffix
+}
+
+ calculate_galasactl_executable
```
Comment (techcobweb) [inline test-galasactl-ecosystem.sh:1333]: I'd be tempted to move this out into a second script.
Then use this:
```
source ${BASEDIR}/that-subscript.sh
```

Then you can invoke properties_test from the bigger file.

**Review by techcobweb — APPROVED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-160-mr9fn for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-160-mr9fn-r-s7mj8 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-160-62bz5 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**